lib/rules/prefer-query-by-disappearance.ts

```diff
@@ -0,0 +1,184 @@
+import { TSESTree } from '@typescript-eslint/experimental-utils';
+
+import { createTestingLibraryRule } from '../create-testing-library-rule';
+import {
+  getPropertyIdentifierNode,
+  isArrowFunctionExpression,
+  isCallExpression,
+  isMemberExpression,
+  isFunctionExpression,
+  isExpressionStatement,
+  isReturnStatement,
+  isBlockStatement,
+} from '../node-utils';
+
+export const RULE_NAME = 'prefer-query-wait-disappearance';
+export type MessageIds = 'preferQueryByDisappearance';
```

Comment:
No need to export these, as they're not used anywhere else

```suggestion
const RULE_NAME = 'prefer-query-wait-disappearance';
type MessageIds = 'preferQueryByDisappearance';
```

Reply:
`RULE_NAME` is being used here: https://github.com/testing-library/eslint-plugin-testing-library/pull/402/files#diff-85beea9fec5d5f9c3167a9bb1ca393a7a5c511cf3d4fec7acbe4a5558c1d1a79R2
But `MessageIds` is not - removed `export`.

Reply:
Am I missing something or is this link the wrong one? 🤔

Reply:
It's the correct link but probably `tests/lib/rules/prefer-query-by-disappearance.test.ts` file is not loading so you need to click `Load diff` to be able to go the linked reference.